client/src/pages/Login/Login.tsx

```diff
@@ -36,25 +36,43 @@ export default function Login(): JSX.Element {
     });
   };
 
+  const [open, setOpen] = React.useState(true);
+
+  const handleOpenLoginModal = () => {
+    setOpen(true);
+  };
+
+  const handleCloseLoginModal = () => {
+    setOpen(false);
+  };
+
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
-              </Grid>
+      <Dialog
```

Comment:
You can add `maxwidth` to both dialogs to try to make them the same width! https://material-ui.com/api/dialog/ It's ok for them to be different heights since they have different amounts of content

Reply:
fixed